MarsRealEstate-Starter/app/src/main/java/com/example/android/marsrealestate/detail/DetailFragment.kt

```diff
@@ -21,7 +21,6 @@ import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import androidx.fragment.app.Fragment
-import androidx.lifecycle.ViewModelProviders
 import com.example.android.marsrealestate.databinding.FragmentDetailBinding
 
```

Comment:
In this file we should also set binding.lifecycleOwner = this, right? Since it appears in the final code

Reply:
Done!